common/beerocks/bwl/CMakeLists.txt

```diff
@@ -76,6 +76,9 @@ if(BWL_TYPE STREQUAL "DWPAL")
 
 elseif(BWL_TYPE STREQUAL "NL80211")
 
+    find_package(nl-genl-3 REQUIRED)
+    list(APPEND BWL_LIBS nl-genl-3)
```

Comment:
We actually also use nl-3 - so like for dwpal, we should do a find_package for it and add it to the libs.

It happens to work because:
- the header files are in the same directory;
- we do dynamic linking, and libnl-genl-3.so pulls in libnl-3.so.

It would not work if we attempted to do static linking.

Reply:
I'll just leave it for another PR.